TimeAgo.js

```diff
@@ -1,51 +1,39 @@
-var React = require('react')
-var ReactNative = require('react-native');
-var moment = require('moment');
-var TimerMixin = require('react-timer-mixin');
+import React, { Component, PropTypes } from 'react';
+import { View, Text } from 'react-native';
+import moment from 'moment';
+import TimerMixin from 'react-timer-mixin';
 
-var { PropTypes } = React;
-var { Text } = ReactNative;
-
-var TimeAgo = React.createClass({
-  mixins: [TimerMixin],
-  propTypes: {
-    time: React.PropTypes.oneOfType([
-      React.PropTypes.string,
-      React.PropTypes.number,
-      React.PropTypes.array,
-      React.PropTypes.instanceOf(Date)
-    ]).isRequired,
-    interval: PropTypes.number,
-    hideAgo: PropTypes.bool
-  },
-
-  getDefaultProps() {
-    return {
-      hideAgo: false,
-      interval: 60000
-    }
-  },
+export default class TimeAgo extends Component {
 
   componentDidMount() {
-    var {interval} = this.props;
-    this.setInterval(this.update, interval);
-  },
+    TimerMixin.setInterval(this.forceUpdate.bind(this), this.props.interval);
+  }
 
   componentWillUnmount() {
-    this.clearInterval(this.update);
-  },
-
-  // We're using this method because of a weird bug
-  // where autobinding doesn't seem to work w/ straight this.forceUpdate
-  update() {
-    this.forceUpdate();
-  },
+    TimerMixin.clearInterval(this.forceUpdate.bind(this));
+  }
 
   render() {
     return (
-      <Text {...this.props}>{moment(this.props.time).fromNow(this.props.hideAgo)}</Text>
-    );
+      <Text {...this.props}>{moment('this.props.time').fromNow(false)}</Text>
```

Comment:
@ashik94vc You've wrapped `this.props.time` in a string, which I don't think you meant to do.

also fromNow is fixing to false, rather than using the props